packages/frontend/app/(dashboard)/course/[cid]/components/QueueInfoColumnButton.tsx

```diff
@@ -1,6 +1,12 @@
 import { cn } from '@/app/utils/generalUtils'
 import { Button, ButtonProps } from 'antd'
-import { PropsWithChildren } from 'react'
+import React, { PropsWithChildren } from 'react'
+
+const JoinZoomButton: React.FC<
+  PropsWithChildren<QueueInfoColumnButtonProps>
+> = (props) => (
+  <QueueInfoColumnButton type="primary" onClick={props.onClick} {...props} />
+)
```

Comment:
With this, do we think we can rename this to something else purely because there is another component named the exact same?

Reply:
Actually wait, after further investigation, I realised that this component actually isn't being used anywhere. The one on the queue info column is using the other JoinZoomButton
![image](https://github.com/user-attachments/assets/011eba16-2e06-4092-aa0e-be66c2c198e7)